docs/hub/spaces-circleci.md

```diff
@@ -0,0 +1,54 @@
+# Managing Spaces with CircleCI Workflows
+
+You can keep your app in sync with your GitHub repository with a **CircleCI workflow**. 
```

Comment:
It might be good to add a one-sentence explanation of CircleCI/CircleCI workflows here for people who are not familiar with it.

Reply:
Great shout @davanstrien 👍🏽 I have added a paragraph with a few lines about CircleCI and a CircleCI workflow.